modules/framework/run-locally.sh

```diff
@@ -172,6 +172,7 @@ function setup_galasa_dev() {
     info "Setting environment variables"
     export GALASA_OBR_VERSION=$GALASA_VERSION
     export GALASA_BOOT_JAR_VERSION=$GALASA_VERSION
+    export GALASA_OWNER_LOGIN_IDS="tester"
```

Comment:
Is "tester" a loginID? Seems to me like a role. If this is just an arbitrary value to start the API server and it doesn't really matter, maybe can it be something more generic like "owner-login-id"?

Reply:
Fixed by adding more users, one of which is 'owner'